src/models/HourlyForecast.tsx

```diff
@@ -0,0 +1,25 @@
+import { Coordinate } from './Coordinate';
+import { City } from './City';
+import { Weather } from './Weather';
+
+export class HourlyForecast {
+
+    City: City;
+    Coord: Coordinate;
+    Code: number;
```

Comment:
What is the main purpose of this `code` attribute? Can we use an `enum` instead of a `number` here?

Reply:
During the time that I was researching inside the OpenWeather API, I saw this attribute only returned as a numerical value. So this is the reason that we are using it as number.